### DISCUSSION TITLE
Fluidd Asking for Login After Firmware Update

### DISCUSSION THREAD

**Discussion by buildjournal** [Guilouz/Creality-Helper-Script-Wiki#620] [Issues]
I installed firmware 1.3.3.29 on my K1 Max and did my normal install procedure using the Helper Script as usual. Previously I've done about 5 firmware updates successfully. **All of a sudden after this update, my Fluidd dashboard is asking for a username and password.** 

When I was researching, a few users said I can turn my `force_logins` to false via moonraker.conf but I wasn't sure how to access that via SSH nor was my Fluidd working to access files. That's when I decided to try Mainsail and oddly enough it worked.

I immediately went to check moonraker.conf to change the force_logins settings, but oddly, it's already set to false.

So I'm confused. Anyone else have this issue? I'd like to regain access to Fluidd.

[authorization]
force_logins: False

**Comment by K20FRITZ** [comment]
Are you using putty? @buildjournal

**Comment by K20FRITZ** [comment]
i used MobaXterm to ssh in and then used the file browser to get to

/usr/data/printer_data/config/moonraker.conf 
<img width="194" alt="moon" src="https://github.com/user-attachments/assets/09002a0a-119a-4627-ba82-1b6754c5e114">
add your ip in the "trusted_clients"
ie 10.10.0.0/16 or what ever your routers ip address is @buildjournal
